packages/dropdown/src/Default/Default.types.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface DropDownItemProps {
+  id: string;
+  label: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any | undefined;
+  length: number;
+}
+
+export interface DropDownContainerProps {
+  label: string;
+  selectedItem: string;
+  items: DropDownItemProps;
+}
+
+export interface DropdownDefaultCommons {
+  // appearance?: 'Primary' | 'Outline';
+  theme?: 'Default' | 'Dark';
+  size?: 'Sm' | 'Md' | 'Lg';
+  prefix?: string;
+  children: React.ReactNode;
+  dropdown: DropDownContainerProps;
+  items: DropDownItemProps;
+}
+
+export type DropdownDefaultProps = Partial<DropdownDefaultCommons> &
+  React.AllHTMLAttributes<HTMLDivElement> &
```

Comment:
The `React.AllHTMLAttributes` is too generic for our component, let's keep only `export type DropdownDefaultProps = DropdownDefaultCommons & bs.DropdownButtonProps;`.

Removing the `Partial<>` will also fix some typescript errors. The [Partial](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) sets all properties as optional, consequently you will see a bunch of `Object is possibly 'undefined'` errors. We don't need to set all `DropdownDefaultCommons` properties as optional.